documentation/modules/ROOT/pages/transformations/event-flattening.adoc

```diff
@@ -304,3 +304,32 @@ When the SMT adds metadata fields to the simplified record's header, it prefixes
  +
 If you specify a field that is not in the change event record, the SMT does not add the field to the header.
 |===
+
+The following table describes the options that you can specify to configure the event flattening SMT to integrate with the event changes SMT.
+
+
+.Descriptions of event flattening SMT configuration options for integration with event changes SMT
+[cols="30%a,25%a,45%a",subs="+attributes",options="header"]
+|===
+|Option
+|Default
+|Description
+
+|[[extract-new-record-state-header-changed-name]]xref:{link-event-flattening}#extract-new-record-state-header-unchanged-name[`header.unchanged.name`]
```

Comment:
```suggestion
|[[extract-new-record-state-header-changed-name]]xref:extract-new-record-state-header-unchanged-name[`header.unchanged.name`]
```
There's no need to include the category attribute for xrefs that link to content in the same file. Same comment applies to the table entries that follow.